src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionRuleEndpointIT.java

```diff
@@ -141,8 +142,12 @@ public void updateCreatedActionRule() throws UnirestException {
   }
 
   private ActionPlanDTO createActionPlan() throws UnirestException {
+    HashMap<String, String> selectors = new HashMap<>();
+    selectors.put("selector_key", "selector_value");
```

Comment:
what is this change?

Reply:
whoops added this in to check what passing objects into `.with()` looks like, will remove.